src/PublicApiGenerator/EventNameBuilder.cs

```diff
@@ -39,20 +39,22 @@ public static class EventNameBuilder
             var addScopeAttributes = addAccessorAttributes & MemberAttributes.ScopeMask;
             switch (addScopeAttributes)
             {
-                case MemberAttributes.Static when addScopeAttributes == MemberAttributes.Static && !isNew.HasValue:
+                case MemberAttributes.Static when isNew.HasValue is false:
```

Comment:
`isNew.HasValue is false` could be `isNew is null`

Reply:
Dude you are a machine.